RockPaperScissors.c

```diff
@@ -0,0 +1,69 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+int main()
+{
+	printf("It's time to play Rock, Paper, Scissors\n");
+	printf("First one to win 3 wins!\n");
+	int uwin = 0, cwin = 0;
+	while (uwin != 3 && cwin != 3) {
+		srand(time(NULL));
+		int r = rand() % 3;
+		char cpu;
+		switch (r) {
```

Comment:
I think you could have reduced this process somehow - since this was how the computer choice was being generated, i'm pretty sure you could have gotten rid of this all together.